features/proxy_config.feature

```diff
@@ -21,8 +21,8 @@ Feature: Proxy configuration
         .*CONNECT contracts.canonical.com.*
         """
         When I run `truncate -s 0 /var/log/squid/access.log` `with sudo` on the `proxy` machine
-        And I configure uaclient `apt_http` proxy to use `proxy` machine
-        And I configure uaclient `apt_https` proxy to use `proxy` machine
+        And I run `ua config set apt_http_proxy=http://<ci-proxy-ip>:3128` with sudo
+        And I run `ua config set apt_https_proxy=http://<ci-proxy-ip>:3128` with sudo
         And I verify `/var/log/squid/access.log` is empty on `proxy` machine
         Then I verify that no files exist matching `/etc/apt/apt.conf.d/90ubuntu-advantage-aptproxy`
```

Comment:
Since we are running `ua config set` now, this will actually create the apt proxy file ant the test will fail

Reply:
Aye, and related: we may need separate tests for `ua config set` vs "edit config file then run `ua refresh config`"